### PR TITLE
Switch --priveleged to --cap-add SYS_NICE in docker calls

### PR DESCRIPTION
**Pull request by KVS85**
According to https://github.com/docker-library/official-images/pull/5902#issuecomment-492020037

### CODE REVIEW

File: helper.linux.fish [hunk 1]
```diff
@@ -223,7 +223,7 @@ function oskar
   checkoutIfNeeded
   and if test "$ASAN" = "On"
     parallelism 2
-    runInContainer --privileged $UBUNTUBUILDIMAGE $SCRIPTSDIR/runTests.fish
+    runInContainer --cap-add SYS_NICE $UBUNTUBUILDIMAGE $SCRIPTSDIR/runTests.fish
   else
```
Comment (fceller) [inline helper.linux.fish]: Is this the correct place? When running ASAN, we need to trace. Unsure if this is managed by "SYS_NICE"

File: helper.linux.fish [hunk 2]
```diff
@@ -242,15 +242,15 @@ function oskarFull
     launchLdapServer
     and if test "$ASAN" = "On"
       parallelism 2
-      runInContainer --net="$LDAPNETWORK$LDAPEXT" --privileged $UBUNTUBUILDIMAGE $SCRIPTSDIR/runFullTests.fish
+      runInContainer --net="$LDAPNETWORK$LDAPEXT" --cap-add SYS_NICE $UBUNTUBUILDIMAGE $SCRIPTSDIR/runFullTests.fish
```
Comment (fceller) [inline helper.linux.fish]: same as above

File: helper.linux.fish [hunk 3]
```diff
@@ -242,15 +242,15 @@ function oskarFull
...
     else
       runInContainer --net="$LDAPNETWORK$LDAPEXT" $UBUNTUBUILDIMAGE $SCRIPTSDIR/runFullTests.fish
     end
     set s $status
   else
     if test "$ASAN" = "On"
       parallelism 2
-      runInContainer $UBUNTUBUILDIMAGE --privileged $SCRIPTSDIR/runFullTests.fish
+      runInContainer $UBUNTUBUILDIMAGE --cap-add SYS_NICE $SCRIPTSDIR/runFullTests.fish
```
Comment (fceller) [inline helper.linux.fish]: same as above

File: jenkins/helper/server.inc [hunk 4]
```diff
@@ -38,14 +38,14 @@ echo
 docker kill $ARANGO_DOCKER_NAME > /dev/null 2>&1 || true
 docker rm -fv $ARANGO_DOCKER_NAME > /dev/null 2>&1 || true
 docker pull $DOCKER_IMAGE
-docker run --privileged $DOCKER_IMAGE arangosh --version
+docker run --cap-add SYS_NICE $DOCKER_IMAGE arangosh --version
```
Comment (fceller) [inline jenkins/helper/server.inc]: Unsure what privileges ae required here

File: jenkins/helper/server.inc [hunk 5]
```diff
@@ -38,14 +38,14 @@ echo
...
 
 OUTDIR="`pwd`/output"
 rm -rf $OUTDIR
 mkdir $OUTDIR
 DOCKER_AUTH=""
 STARTER_AUTH=""
-DOCKER_CMD="docker run --privileged --name $ARANGO_DOCKER_NAME -d $EXPORT_PORTS -v $OUTDIR:/testrun"
+DOCKER_CMD="docker run --cap-add SYS_NICE --name $ARANGO_DOCKER_NAME -d $EXPORT_PORTS -v $OUTDIR:/testrun"
```
Comment (fceller) [inline jenkins/helper/server.inc]: Unsure what privileges ae required here

Comment (KVS85) [reply jenkins/helper/server.inc]: Only those which will enable NUMA. I thought we agreed to use it by default.

**Pull request merged**